packages/Tag/src/Tag.js

```diff
@@ -4,41 +4,45 @@ import RawButton from "@paprika/raw-button";
 import useI18n from "@paprika/l10n/lib/useI18n";
 import * as sc from "./Tag.styles";
 import * as types from "./types";
+import ClearIcon from "@paprika/icon/lib/Times";
```

Comment:
absolute import path should be added before relative import path

Reply:
also just curious, why not use `TimesCircle ` ? `TimesCircle ` will be much easier to override the color, since the circle and the X are both inside the SVG path
cc @mikrotron

Reply:
Because of the colour inside of X, it does not seem the same as in the mockup. The X in the mockup is kind of transparent, so the colour is the same as the background colour, but the TimesCircle X is white.

Reply:
oh I thought the TimesCircle's X is transparent? 
![image](https://user-images.githubusercontent.com/38733362/139966504-d51d733d-0317-4bbd-944d-11d3e5e39dff.png)
https://acl-icon-font.herokuapp.com/?path=/story/wasabicons--icons